docs/Tutorials/Python.md

```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
 with the command `. BUILD_DIR/bin/LoadPython.sh`.
+
```

Comment:
Could you add this file to the `Tutorials` page? Currently if I go to `User Tutorials` it doesn't show up, but given its location I think this was an oversight when it was added.